lib/dialogs/share/index.tsx

```diff
@@ -207,9 +205,9 @@ export class ShareDialog extends Component {
   }
 }
 
-export default connect(
-  state => ({
-    settings: state.settings,
-  }),
-  { updateNoteTags }
-)(ShareDialog);
+const mapStateToProps = ({ settings, ui: { note } }) => ({
+  settings,
+  note: note || {},
```

Comment:
given that we know this will be `T.NoteEntity | null` do we need the `|| {}` here? I'm guessing that other code in this file was easier to keep if we defaulted to an empty object, but at the same time `{}` seems more obscure than `null` to indicate that it's missing.

don't worry about it if we're matching existing behavior or this is the difference between a small change and a huge change. it would be nice though if we let those types flow smoothly through the module.

Reply:
I did that for a reason however I don't think we need the empty object.